rules/windows/process_creation/proc_creation_win_susp_filefix_execution_pattern.yml

```diff
@@ -0,0 +1,74 @@
+title: Suspicious FileFix Execution Pattern
+id: 4be03877-d5b6-4520-85c9-a5911c0a656c
```

Comment:
If it's changed by this much, obsolete the old rule and create this as a new one that obsoletes the other. Otherwise remember `git mv` :)